snapshot/src/test/java/com/schibsted/snapshot/WarpComponentsTest.kt

```diff
@@ -65,6 +66,18 @@ class SnapshotTest(
             }
         }
     }
+
+    @Test
```

Comment:
It would be better to have the checkbox tests in a separate class/file. Also please include all the different states and styles of the checkbox